src/components/EntityInfo/EntityHeader.vue

```diff
@@ -9,15 +9,26 @@
       <!-- Subtitle -->
       <div>
         <span id="business-description">{{ businessDescription }}</span>
+        <span id="active-util" class="ml-3 pl-3" v-if="isInLimitedRestoration">
+          Active until {{ getLimitedRestorationActiveUntil || 'Unknown' }}
+        </span>
+      </div>
 
-        <span id="limited-restoration" class="ml-3" v-if="isInLimitedRestoration">
-          <span class="ml-3">Active until {{ getLimitedRestorationActiveUntil || 'Unknown' }}</span>
-          <v-chip class="primary mt-n1 ml-3 pointer-events-none font-weight-bold"
+      <div id="historical" class="mt-2" v-if="isHistorical">
+          <v-chip
```

Comment:
Can you fix indentation here please (2 spaces instead of 4)? Also, the lines below inside this div tag.

Reply:
Fixed, thanks!